src/Sulu/Component/Rest/ListBuilder/AbstractListBuilder.php

```diff
@@ -238,8 +238,19 @@ public function search($search)
      */
     public function sort(AbstractFieldDescriptor $fieldDescriptor, $order = self::SORTORDER_ASC)
     {
-        $this->sortField = $fieldDescriptor;
-        $this->sortOrder = $order;
+        $this->sortFields = array($fieldDescriptor);
```

Comment:
because it was the previous behavior. it also wouldn't be possible to overwrite the sort value. 

for the where example:  there is no andWhere (like in the QueryBuilder, where 'where' is actually overriding the last set value).

probably the naming of multiple methods of the listbuilder should be reconsidered. since it's not always clear, what the functionality is